src/adt/tests.rs

```diff
@@ -1,10 +1,10 @@
 use anyhow::Result;
 use elementtree::Element;
 
-use super::foundation::Color;
-use super::svg::Canvas;
 use super::background::Background;
+use super::foundation::Color;
 use super::image::Image;
+use super::svg::Canvas;
 
 pub fn compare_svg_text(left: &str, right: &str) -> Result<()> {
```

Comment:
Looks like we are never going to use `compare_svg_text`. I am going to remove it at some point.